portfolio/src/main/webapp/request.js

```diff
@@ -115,14 +145,34 @@ function addScheduledTaskToDom(scheduledTaskWrapper, resultElement) {
   const cardTitle = document.createElement('h4');
   cardTitle.classList.add('card-title');
   cardTitle.innerText = taskName;
+  cardTitle.setAttribute('data-task-name', taskName);
   cardBody.appendChild(cardTitle);
 
   const timeText = document.createElement('p');
   timeText.classList.add('card-text');
-  timeText.innerText = 'Scheduled for: ' + taskDate;
+  timeText.innerText = SCHEDULED_TIME_INITIAL_TEXT + taskDate;
+  timeText.setAttribute('data-task-time', taskDate);
   cardBody.appendChild(timeText);
 
-  const eventList = document.getElementById('scheduled-task-list');
-  eventList.innterHTML = '';
-  eventList.appendChild(newResultCard);
+
+  const durationText = document.createElement('p');
+  durationText.classList.add('card-text');
+  durationText.innerText = DURATION_INITIAL_TEXT + taskDurationMinutes;
+  durationText.setAttribute('data-task-duration', taskDurationMinutes);
```

Comment:
Consider renaming the attribute name to data-task-duration-minutes so the time unit is clear.

Reply:
Done.